scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -688,17 +688,16 @@ def _response_constraints_validation(self, response, result):
             if 'constraints' not in value:
                 continue
 
-            expected_name = 'value'
             received_value = response.get('value')
             if not self.is_attribute:
                 expected_name = value.get('name')
                 if received_value is None or expected_name not in received_value:
-                    result.error(check_type, error_name_does_not_exist.format(
-                        name=expected_name))
-                    continue
-
-                received_value = received_value.get(
-                    expected_name) if received_value else None
+                    # It is possible we are checking for constraint where
```

Comment:
There is a problem here I still need to sort out that I just thought about. I need to make sure that when there is not hasValue constraint that we still fail here appropriately

Reply:
This isn't a real issue, but it may help with verbosity of the error message so I have added a fix here